doc/api/readline.md

```diff
@@ -628,6 +628,13 @@ the best compatibility if it defines an `output.columns` property and emits
 a `'resize'` event on the `output` if or when the columns ever change
 ([`process.stdout`][] does this automatically when it is a TTY).
 
+In order to end the wait for the `stdin` input stream (so that the program
+does not appear to have hanged), `unref` function can be used:
```

Comment:
I think it would be helpful to indicate why does the program hangs. What do you think of this:
```suggestion
When creating an `Interface` using `stdin` as input stream, the program will not
terminate until it receives `EOF` (<kbd>Ctrl</kbd>+<kbd>D</kbd>). If you want
your application to exit without waiting for user input, you can `unref` the
standard input:
```

Reply:
I like your suggestion, thanks!